app/controllers/requests_controller.rb

```diff
@@ -130,4 +130,37 @@ def request_params
                                     :port, :application_name, :description, :comment,
                                     :rejection_information, :status, user_ids: [], sudo_user_ids: [])
   end
+
+  def puppet_node_script(request)
+
+    puppet_string =
+'class node_vm-%s {
```

Comment:
The example files given by the admin were different than the US.
```suggestion
'class node_$%s {
```